### PR TITLE
implemented runtime attribute assignment in with_attrs

### PR DESCRIPTION
**Pull request by huard**
Related to #41

### CODE REVIEW

File: xclim/indices.py [hunk 1]
```diff
@@ -534,6 +566,8 @@ def tx_mean(tasmax, freq='YS'):
     return arr.mean(dim='time')
 
 
+@with_attrs(standard_name='tx_min', long_name='Minimum of daily maximum temperature',
+            cell_methods='time: minimum within {freq}')
```
Comment (Zeitsperre) [inline xclim/indices.py]: I believe this is the source of Python2 errors. I don't think that fstrings with keywords can be passed within function arguments. Do you know of a workaround?

Comment (Zeitsperre) [reply xclim/indices.py]: Confirming this. Hard coding 'years' in `cell_methods` passes py27 build.

**Comment by huard** [comment]
But is the general solution proposed fixing the issue you had in mind ?

**Comment by Zeitsperre** [comment]
I think so. I just need to implement the logger using the module you pointed out but for this issue I feel that this manages to maintain/change attrs as needed and isn' clunky boilerplate code. If the py27 build can be worked around, this would effectively close #41 and #43.

**Comment by huard** [comment]
I think this can be merged.

**Pull request merged**